docs/guides/debugging.md

```diff
@@ -922,3 +922,33 @@ latter, the following commands might be of help:
     scylla generate-object-graph # generates and visualizes an object graph
 
 Good luck, you are off the charted path here.
+
+#### Use python for inspecting objects and performing computations
+GDB is well integrated with Python, and besides using the predefined set of commands from `scylla-gdb.py`, you can also run an interactive prompt by typing `python-interactive` or `pi` in gdb console. That allows you to benefit from the rich set of helper classes and wrappers defined in `scylla-gdb.py`, e.g. in order to iterate over one of the supported collection types (vectors, small vectors, maps, tuples, etc.) and print only the interesting bits.
+
+Example:
+```gdb
+(gdb) source /path/to/scylla-gdb.py
+(gdb) python-interactive
+>>> db = gdb.parse_and_eval("*(seastar::sharded<replica::database>*)debug::the_database")
+>>> instances = std_vector(db["_instances"])
+>>> for i, instance in enumerate(instances):
+...     print(i, instance["service"])
```

Comment:
Unrelated note: we have a `$sharded_local()` convenience function just for this ;)